### ISSUE TITLE
How to do draw math equation using mXparser?

### ISSUE THREAD

**Issue by syaifulnizamyahya** [mariuszgromada/MathParser.org-mXparser#272]
I want to draw math equation like how Latex draw math equation. 
What library to use?

**Comment by mariuszgromada** [comment]
I am not sure if I understand the question correctly. Do you mean computing the value of a function / equation for multiple arguments and then plotting the chart of that function?

**Comment by syaifulnizamyahya** [comment]
Hi @mariuszgromada Sorry for my confusing question. Let me rephrase.
Lets say I have this Expression :
Expression e = new Expression("2-(32-4)/(23+4/5)-(2-4)*(4+6-98.2)+4");

I want to display it like this :
![image](https://user-images.githubusercontent.com/10744241/187118597-7beb30f5-259d-48dd-8f7d-856625d09156.png)

(Taken from https://mathparser.org/mxparser-tutorial/simple-expressions/)

Actually, what I want to do is simple math exercise multiplatform apps for my kids. I plan to use MAUI. 
Thanks.

**Comment by mariuszgromada** [comment]
Ok now I understand. I'm using LaTeX on the website. The mXparser software is not intended to display an expression in a graphical-mathematical form. In the future I will consider adding a functionality in mXparser that generates LaTeX code based on the expression string given in the mXparser syntax.
